src/sentry/rules/filters/issue_severity.py

```diff
@@ -1,76 +0,0 @@
-from collections import OrderedDict
-from typing import Any
-
-from django import forms
-
-from sentry import features
-from sentry.eventstore.models import GroupEvent
-from sentry.issues.grouptype import GroupCategory
-from sentry.models.group import Group
-from sentry.rules import EventState, MatchType
-from sentry.rules.filters import EventFilter
-from sentry.types.condition_activity import ConditionActivity
-
-SEVERITY_MATCH_CHOICES = {
-    MatchType.GREATER: "greater than",
-    MatchType.GREATER_OR_EQUAL: "greater than or equal to",
-    MatchType.LESS_OR_EQUAL: "less than or equal to",
-    MatchType.LESS: "less than",
-}
-CATEGORY_CHOICES = OrderedDict([(f"{gc.value}", str(gc.name).title()) for gc in GroupCategory])
-
-
-class IssueSeverityForm(forms.Form):
-    value = forms.NumberInput()
-
-
-class IssueSeverityFilter(EventFilter):
-    id = "sentry.rules.filters.issue_severity.IssueSeverityFilter"
```

Comment:
Are there any existing rules using this filter which need to be cleaned up?

Reply:
nope, I cleaned up these - there were only two, both for the `sentry` project